example/src/main/tut/data/Kleisli.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: docs
+title:  "Kleisli"
+---
+
+# Kleisli
+
+Given a given `Category` C, and a `Monad` M over C, we can construct a new Category, called the `Kleisli Category` of M, whose objects are those of C, but whose morphisms are those arrows `A => M[B]`. The arrows of this new category are called `Kleisli Arrows`, or, more succinctly, `Kleisli`s.
+
+
+## Basics
+
+While this may seem like a complicated topic, in practice, the notion of a `Kleisli` is simple. Any function `A => M[B]` is a `Kleisli`. The `Kleisli` data object provides a succinct newtype wrapper for easy costless conversion between `Kleisli` and its corresponding function type via two special functions, `wrap`, and `run`:
```

Comment:
`unwrap` sounds more natural to be the opposite of `wrap`

Reply:
It's convention to use `wrap` and `run`. I'm not attached to any specific set of names, so if people agree, we'll change it. We can also bikeshed about names at any point during or even after this PR